website/content/docs/platform/k8s/vso/installation.mdx

```diff
@@ -24,26 +24,69 @@ HashiCorp helm repository and ensure you have access to the chart:
 
 ```shell-session
 $ helm repo add hashicorp https://helm.releases.hashicorp.com
+
 "hashicorp" has been added to your repositories
+```
 
+```shell-session
 $ helm search repo hashicorp/vault-secrets-operator
+
 NAME           	CHART VERSION	APP VERSION	DESCRIPTION
 hashicorp/vault-secrets-operator	0.3.1       	0.3.1     	Official HashiCorp Vault Secrets Operator Chart
 ```
 
 Then install the Operator:
 
 ```shell-session
-$ helm install --create-namespace --namespace vault-secrets-operator vault-secrets-operator hashicorp/vault-secrets-operator --version 0.3.1
+$ helm install --version 0.3.1 --create-namespace --namespace vault-secrets-operator vault-secrets-operator hashicorp/vault-secrets-operator
 ```
 
 
 ## Upgrading using helm
 
-Upgrades may be performed with `helm upgrade` on an existing install. Please
+Upgrades may be performed with `helm upgrade` on an existing installation. Please
```

Comment:
```suggestion
You can upgrade on an existing installation with the `helm upgrade` . Please
```

Style correction: write in active voice, address the reader when providing instruction

Reply:
Fixed in https://github.com/hashicorp/vault/pull/23512/commits/f088661672eb9de9cea37ec46f670c893fdebd7d